electricitymap/contrib/capacity_parsers/EMBER.py

```diff
@@ -0,0 +1,182 @@
+import io
+from datetime import datetime
+from logging import getLogger
+
+import pandas as pd
+import pycountry
+from bs4 import BeautifulSoup
+from requests import Response, Session
+
+from electricitymap.contrib.config import ZoneKey
+
+""" Collects capacity data from the yearly electricity data from Ember. The data and documentation can be found here: https://ember-climate.org/data-catalogue/yearly-electricity-data/"""
+logger = getLogger(__name__)
+EMBER_VARIABLE_TO_MODE = {
+    "Bioenergy": "biomass",
+    "Coal": "coal",
+    "Gas": "gas",
+    "Hydro": "hydro",
+    "Nuclear": "nuclear",
+    "Other Fossil": "unknown",  # mostly oil it seems
+    "Other Renewables": "unknown",
+    "Solar": "solar",
+    "Wind": "wind",
+}
+EMBER_URL = "https://ember-climate.org"
+SOURCE = "Ember, Yearly electricity data"
+SPECIFIC_MODE_MAPPING = {
+    "BD": {"Other Fossil": "oil"},
+    "CO": {"Other Fossil": "oil"},
+    "CY": {"Other Fossil": "oil"},
+    "KR": {"Other Fossil": "oil"},
+    "KW": {"Other Fossil": "oil"},
+    "MN": {"Other Fossil": "coal"},
+    "SG": {"Other Fossil": "coal"},
+    "SV": {"Other Fossil": "oil"},
+    "TR": {"Other Fossil": "oil", "Other Renewables": "geothermal"},
+    "TW": {"Other Fossil": "oil"},
+    "ZA": {"Other Fossil": "oil"},
+}
+
+EMBER_ZONES = [
+    "AR",
+    "AW",
+    "BA",
+    "BD",
+    "BO",
+    "BY",
+    "CO",
+    "CR",
+    "CY",
+    "DO",
+    "GE",
+    "GT",
+    "HN",
+    "KR",
+    "KW",
+    "MD",
+    "MN",
+    "MT",
+    "MX",
+    "NG",
+    "NZ",
+    "PA",
+    "PE",
+    "RU",
+    "SG",
+    "SV",
+    "TH",
+    "TR",
+    "TW",
+    "UY",
+    "ZA",
+]
+
+
+def map_variable_to_mode(data: pd.Series) -> str:
+    data = data.copy()
+    zone = data["zone_key"]
+    variable = data["variable"]
+    if zone in SPECIFIC_MODE_MAPPING:
+        if variable in SPECIFIC_MODE_MAPPING[zone]:
+            mode = SPECIFIC_MODE_MAPPING[zone][variable]
```

Comment:
```suggestion
            return SPECIFIC_MODE_MAPPING[zone][variable]
```
One final code style suggestion would be to reduce the amount of else, by directly returning :)